_extension/package.json

```diff
@@ -64,21 +70,21 @@
     },
     "main": "./dist/extension.js",
     "files": [
-        "dist"
+        "dist",
+        "lib",
+        "LICENSE",
+        "logo.png"
     ],
     "scripts": {
         "build": "tsc",
         "watch": "tsc --watch",
-        "build:prod": "esbuild src/extension.ts --bundle --external:vscode --platform=node --format=cjs --outfile=dist/extension.js --minify",
-        "package": "vsce package --skip-license --no-dependencies --out typescript-lsp.vsix",
-        "install-extension": "code --install-extension typescript-lsp.vsix",
-        "vscode:prepublish": "npm run build:prod"
+        "bundle": "esbuild src/extension.ts --bundle --external:vscode --platform=node --format=cjs --outfile=dist/extension.js --minify"
```

Comment:
Not introduced in this PR, but I noticed that `dist/extension.js` clobbers the one built by tsc and breaks debugging.

Reply:
Yeah, I could fix that now that I'm copying the extension after bundling to produce each platform.

Reply:
Fixed it.